src/webgl/lineFeature.js

```diff
@@ -25,8 +25,10 @@ var flagsLineJoin = {  // uses 3 bits with flagsLineCap
   round: 6,
   'miter-clip': 7
 };
-var flagsNearLineShift = 2, flagsFarLineShift = 5;
-var flagsNearOffsetShift = 8;  // uses 11 bits
+var flagsNearLineShift = 2, flagsNearLineMult = 1 << flagsNearLineShift;
```

Comment:
Same here, if we add some code comment on how these flags are used, it would be be easy to remember and understand the code

Reply:
@aashish24 I've added more to the comment above all of the flag definitions.